django/contrib/admin/options.py

```diff
@@ -468,7 +468,7 @@ def has_add_permission(self, request):
         """
         opts = self.opts
         codename = get_permission_codename('add', opts)
-        return request.user.has_perm("%s.%s" % (opts.app_label, codename))
+        return request.user.has_perm('%s.%s' % (opts.app_label, codename))
```

Comment:
Can you pull these cosmetic changes into a separate commit please? 
(I know they go with the _Patch style_ guides, but they're not related to the issue here.)

Reply:
Should I put the changes in a separate commit or do them in a separate PR?